src/SFA.DAS.ApplyService.Web/Services/QuestionPropertyTokeniser.cs

```diff
@@ -23,7 +23,9 @@ public async Task<string> GetTokenisedValue(Guid applicationId, string tokenised
                 return string.Empty;
             }
 
-            if (ContainsTokens(tokenisedValue))
+            int maxCounter = 5;
+            int startCounter = 0;
+            while (ContainsTokens(tokenisedValue) && startCounter < maxCounter)
```

Comment:
I think there's a better way of doing this but it'll do for now. Maybe raise as tech debt

Reply:
maybe, but I'm wary of a while loop that assesses a value altered within it, without some maximum counter.  That's how entire apps crash.....